src/quantuminspire/qiskit/backend_qx.py

```diff
@@ -282,36 +284,76 @@ def _collect_measurements(experiment):
             measurements = [[index, index] for index in range(number_of_qubits)]
         return {'measurements': measurements, 'number_of_clbits': number_of_clbits}
 
+    @staticmethod
+    def __qubit_2_classical_hex(qubit_register, measurements, number_of_qubits):
```

Comment:
Then, if there is no longer a need for a special __convert_histogram, and __get_memory data, this can be combined into one `__convert_results`?

Reply:
or retrieve_memory_and_counts or whatever.

Reply:
I agree when the previous comment is solved